assignments/objects.js

```diff
@@ -18,31 +18,66 @@ const example = {
 }
 
 // Write your intern objects here:
+const interns = [{
+  "id": 1,
+  "email": "mmelloy0@psu.edu",
+  "name": "Mitzi",
+  "gender": "F"
+},
+{
+  "id": 2,
+  "email": "kdiben1@tinypic.com",
+  "name": "Kennan",
+  "gender": "M"
+  "speak": function() {
+    console.log("Hello my name is")
+  }
+},
+{
+  "id": 3,
+  "email": "kmummery2@wikimedia.org",
+  "name": "Keven",
+  "gender": "M"
```

Comment:
put a comma `,` after gender to be able to access your speak method. You can then finish it with a string template literal to call it down below and use a similar method for the multiply nums functions. 

Kind of like this ``Hello my name is ${this.name}``